### ISSUE TITLE
ObservableProperty approaches

### ISSUE THREAD

**Issue by KathiSchleidt** [w3c/sdw-sosa-ssn#52]
@rob-metalinkage `noted that there are 3 different ways to describe observed property (?) in SOSA - and there are examples of all 3 methods deployed in the world; he wondered whether these 3 methods could be formalised as profiles? ` in w3c/sdw#1402 

Going through the [examples](https://www.w3.org/TR/vocab-ssn/#examples), I see quite a mixed bag of options:

**Reusable ObservableProperty** 
E.g., example [B.2 Coal Oil Point Reserve](https://www.w3.org/TR/vocab-ssn/#coil_oil_plant-sosa)
```
 <RelativeHumidity> a sosa:ObservableProperty ;
   rdfs:comment "Humidity is a measure of the moisture content of air."@en ;
   rdfs:label "Relative Humidity"@en .
```

**ObservableProperty linked to FoI**
E.g., examples [B.3 apartment 134](https://www.w3.org/TR/vocab-ssn/#apartment-134) and [B.4 Tree height measurement](https://www.w3.org/TR/vocab-ssn/#tree-height)

```
<apartment/134/electricConsumption> rdf:type sosa:ObservableProperty ;
  sosa:isObservedBy <sensor/926>  .
```

```
<tree/124#height>  rdf:type    sosa:ObservableProperty , ssn:Property ;
  rdfs:label "the height of tree w3c/sdw#124"@en ;
  ssn:isPropertyOf <tree/124> .
```
_Note: the tree height example still has an error at the end, closes wrong tree_

**ObservableProperty linked to FoI and Location**
E.g., example [B.11 IP68 Smart Sensor](https://www.w3.org/TR/vocab-ssn/#ip68-smart-sensor)

**ObservableProperty only referenced in Observation**
E.g., example [B.6 Number of sunspots](https://www.w3.org/TR/vocab-ssn/#sunspots)

```
<Observation/7536> rdf:type sosa:Observation ;
  sosa:observedProperty  <Sun#sunspotNumber> ;
  sosa:hasSimpleResult 66 ;
  sosa:phenomenonTime [
    rdf:type time:Instant ;
    time:inXSDDateTimeStamp "2017-03-31T11:51:42+00:00"^^xsd:dateTimeStamp ] ;
  sosa:resultTime "2017-03-31T12:00:00+00:00"^^xsd:dateTimeStamp .
```

`<Sun#sunspotNumber>` never defined, but tightly linked to <Sun>

**ObservableProperty defined by Sensor:observes**
E.g., example [B.11 IP68 Smart Sensor](https://www.w3.org/TR/vocab-ssn/#ip68-smart-sensor)

```
<Sensor/SL-T-P1> a gr:ProductOrService, sosa:Sensor , seas:LoRaCommunicationDevice , <IP68_Outdoor_Temperature_Sensor> ;
    gr:hasBrand [ a gr:Brand ; gr:name "Sensing Labs"@en ] ;
    geo:alt 100.0 ;
    geo:lat 45.75 ;
    geo:lon 4.85 ;
    ssn:implements <IP68_Outdoor_Temperature_Sensor#temperatureSensingProcedure> ;
    ssn:implements <IP68_Outdoor_Temperature_Sensor#batterySensingProcedure> ;
    ssn:observes <Sensor/SL-T-P1#battery> ;
    ssn:observes <Air?lat=45.75&long=4.85#temperature> . 
``` 

and
```

<Observation/5872357#battery> a sosa:Observation ;
    sosa:hasSimpleResult "73.2 %"^^cdt:ucum ;
    sosa:madeBySensor <Sensor/SL-T-P1> ;
    sosa:hasFeatureOfInterest <Sensor/SL-T-P1> ;
    sosa:observedProperty <Sensor/SL-T-P1#battery> ;
    sosa:resultTime "2017-06-20T21:49:18+00:00"^^xsd:dateTime .
```

But <Sensor/SL-T-P1#battery> is never defined

**Comment by dr-shorthair** [comment]
Also see #49

**Comment by maximelefrancois86** [comment]
I see only two ways:

Approach "generic": reusable ObservableProperty , vs Approach "specific": ObservableProperty linked to FoI.
 
The third (ObservableProperty linked to FoI and Location)  is actually a sub-case of Approach "specific", where the FoI is a geo-localized sample of the feature of interest `<Air>`.

As an alternative to defining _profiles_ which would limit the interoperability between these two approaches, one could simply disambiguate the term as proposed in #106

**Comment by dr-shorthair** [comment]
Is this resolved by #126 - see https://w3c.github.io/sdw-sosa-ssn/ssn/#SOSAProperty-instances

**Comment by KathiSchleidt** [comment]
Confirmed, #126 fixes this
